.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Please include a summary of the change, motivation and context
+
+- **on a feature**: describe the feature and how this change fits in it, e.g. this PR makes kafka message.max.bytes configurable to better support batching
+- **on a refactor**: describe why this is better than previous situation e.g. this PR changes logic for retry on healthchecks to avoid false positives
+- **on a bugfix**: link relevant information about the bug (github issue or slack thread) and how this change solves it e.g. this change fixes #99999 by adding a lock on read/write to avoid data races.
+
+### Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Docker-compose
```

Comment:
Why do we need these bullet points? The CI should verify the code against all deploying methods what we have, we cannot rely on users telling us "the PR works on my machine".

Reply:
makes sense but we don't have CI for helm yet so that thing is still a concern. That won't be an issue in this repo but in individual repo that can cause issue sometimes as we faced recently with `raw-spans-grouper` . But that will be solved with CI for helm!

Reply:
Then make it a requirement to run the helm chart locally. The docker compose here is redundant since it runs on CI